core/metrics/insights.go

```diff
@@ -153,6 +154,13 @@ var staticData = sync.OnceValue(func() insights.Data {
 	data.Build.Settings, data.Build.GoVersion = buildInfo()
 	data.OS.Containerized = consts.InContainer
 
+	// Install info
+	packageFilename := filepath.Join(filepath.Dir(conf.Server.ConfigFile), "package")
```

Comment:
I think this is better to be put in the `DataFolder`, as it is a more _permanent_ location. The `ConfigFile` can be moved, and in some packagings it may(?) be stored in a read only folder.

Also, I think the name (`.package`) should be in a constant in the `consts.go` file.

Reply:
Hmm, the intent here is to put it in the installation folder i.e. C:\Program Files\Navidrome which under the msi is the same place but you are right that it doesn't work generically. The data folder doesnt quite seem right either as its not userdata. Does "in the same directory as the executable" work for you instead?

Reply:
I don't think it makes sense to put it in the executable dir, as it can be something like `/usr/bin` and we don't want extraneous, non-executable files in this folder.

I agree it is not userdata, but given the options I think this is the more appropriate place.

Reply:
Yeah, this is definitely not a perfect 1 size fits all situation at all.